.bbtravis.yml

```diff
@@ -187,6 +190,10 @@ script:
     condition: TESTS == "smokes"
     cmd: ./common/smokedist.sh
 
+  - title: container tests
+    condition: TESTS == "container"
+    cmd: ./container/test.sh
```

Comment:
good try, but our workers cannot use docker.

We use docker-hub to check the docker images are building correctly:
https://hub.docker.com/r/buildbot/buildbot-master/builds/

This is post-commit however.